src/Microsoft.Azure.SignalR.Serverless.Protocols/Constants.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols
+{
+    internal static class ServerlessProtocolConstants
```

Comment:
>ServerlessProtocolConstants [](start = 26, length = 27)

`PingMessage` is missed here. Not supported?

Reply:
No. Ping message is totally handled by RT.